### PR TITLE
os/fs: Fix Function Description

### PR DESCRIPTION
**Pull request by KimDongEon**
Fix duplicate spaces

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c90cde18cf090b0995f7f5544ad4c4fd9ef64328] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c90cde18cf090b0995f7f5544ad4c4fd9ef64328] - Code Rule Check OK.

File: os/fs/smartfs/smartfs_utils.c [hunk 1]
```diff
@@ -1829,7 +1829,7 @@ int smartfs_examine_sector(struct smartfs_mountpt_s *fs, char *validsectors, int
 /****************************************************************************
  * Name: smartfs_recover
  *
- * Description: Recovery after  a power failure
+ * Description: Recovery after a power failure
```
Comment (junmin-kim) [inline os/fs/smartfs/smartfs_utils.c]: How about replacing 'Recovery' with 'Recover'?
Other description starts with a verb.

Comment (Taejun-Kwon) [reply os/fs/smartfs/smartfs_utils.c]: ok, please do some more at the volley, add more information (recovery what? ; sector which has broken link)

Comment (KimDongEon) [reply os/fs/smartfs/smartfs_utils.c]: Modify. Thank you for your feedback.

**Comment by seinfra** [comment]
Target : [73c4ffb77d6ccb0b3974eb65f880dd5c6528d11e] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [73c4ffb77d6ccb0b3974eb65f880dd5c6528d11e] - Code Rule Check OK.

**Pull request merged**